### PR TITLE
Restore code that has been lost during modern-ui merge.

### PR DESCRIPTION
**Pull request by badfiles**

### CODE REVIEW

**Comment by syncguru** [comment]
👍

File: account_page.php [hunk 1]
```diff
@@ -191,7 +191,7 @@
 			} ?>
 			<tr>
 				<td class="category">
-					<span class="required"><?php if( $t_force_pw_reset ) { ?> * <?php } ?></span> <?php echo lang_get( 'password' ) ?>
+					<span class="required"><?php if( $t_force_pw_reset ) { ?> * <?php } ?></span> <?php echo lang_get( 'new_password' ) ?>
```
Comment (vboctor) [inline account_page.php]: This doesn't seem right.  This should be password.

Comment (vboctor) [inline account_page.php]: Nevermind, I was looking at a slightly older code.

**Comment by vboctor** [comment]
Merged via 0be20bdbe6bef6ff2a3317e43c8d075c98878aa4

**Pull request closed without merging**